### DISCUSSION TITLE
Is it only for one OBD adapter?

### DISCUSSION THREAD

**Discussion by schpuntik** [jay-oswald/ha-wican#16] [General]
Is it only for one OBD adapter?
What if I use multiple adapters? Will there be support for multiple adapters?

**Comment by jay-oswald** [comment]
@schpuntik yes, multiple adaptors should work, but is not properly tested, HA handles each one getting set up, so it should "just work", I'd need to set up a 2nd WiCAN first to test it myself though

**Comment by jwiesel** [comment]
Hi @schpuntik , as mentioned by jay-oswald you can use multiple WiCAN devices with the integration.

On the device page and integration page they will for now both show up as "WiCAN", but the entities have unique ids.
I already raised the question to give a WiCAN device a friendly name and expose it via the API here: https://github.com/meatpiHQ/wican-fw/issues/228#issuecomment-2661504276

**Comment by jay-oswald** [comment]
@schpuntik @jwiesel I have tested this since getting my pro, and the 2 devices work perfectly fine togeher. I just renamed the device in home assistant, so I don't think giving it a unique name inherintly is that much of a priority

**Comment by Ken7382** [comment]
I've been running two adapters for probably 9 months now, works great! I have both my Impala and Tahoe showing gas gauges and battery levels in HA.
